lib/Doctrine/ORM/Exception/ORMException.php

```diff
@@ -4,11 +4,11 @@
 
 namespace Doctrine\ORM\Exception;
 
-use Doctrine\ORM\ORMException as BaseORMException;
+use Throwable;
 
 /**
  * Should become an interface in 3.0
  */
-class ORMException extends BaseORMException
+interface ORMException extends Throwable
```

Comment:
Extending `Throwable` can have ugly side-effects on PHP 7.3 and below. I'd suggest that we bump to 7.4 at least, so we don't have to deal with them.

https://3v4l.org/rmkAc

Reply:
Fully agree, let's do that.